dotnet/src/webdriver/BiDi/EventHandler.cs

```diff
@@ -21,7 +21,7 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace OpenQA.Selenium.BiDi.Communication;
+namespace OpenQA.Selenium.BiDi;
 
 public abstract class EventHandler(string eventName, IEnumerable<BrowsingContext.BrowsingContext>? contexts = null)
```

Comment:
This `EventHandler` can cause naming conflicts with `System.EventHandler`. I guess it cannot be made internal. So it would be good to rename it to `BiDiEventHandler` or whatever.

Reply:
I made it internal. Thanks for pointing.